docs/api/python/ndarray/ndarray.md

```diff
@@ -606,6 +606,8 @@ The `ndarray` package provides several classes:
     sign
     gamma
     gammaln
+	norm_logcdf
+	norm_derivlogcdf
```

Comment:
indentation

Reply:
About the name: logcdf is too unspecific.
logcdf_norm, derivlogcdf_norm would be OK.

Was "inspired" by scipy.stats.norm.logcdf.